### PR TITLE
Fluidstack catalog

### PR DESCRIPTION
**Pull request by mjibril**
[FluidStack] 

* Add auto updater for FluidStack Catalog
* Add FluidStack Catalog

### CODE REVIEW

File: .github/workflows/update-fluidstack-catalog.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+name: "update-fluidstack-catalog"
```
Comment (Michaelvll) [inline .github/workflows/update-fluidstack-catalog.yml:1]: Let's keep the auto updater in another PR, so we can get the catalog merged first. : )
Our merging order should be:
1. catalog
2. main PR in skypilot repo
3. auto updater

File: catalogs/v5/fluidstack/vms.csv [hunk 2]
```diff
@@ -0,0 +1,67 @@
+InstanceType,AcceleratorName,AcceleratorCount,vCPUs,MemoryGiB,Price,Region,GpuInfo,SpotPrice
+rec3HRxyysDOzxikh,RTXA6000,4,64.0,238.0,6.244061073300001,norway_4_eu,"{'Gpus': [{'Name': 'RTXA6000', 'Manufacturer': 'NVIDIA', 'Count': 4, 'MemoryInfo': {'SizeInMiB': 49152}}], 'TotalGpuMemoryInMiB': 196608}",
```
Comment (Michaelvll) [inline catalogs/v5/fluidstack/vms.csv:2]: Are these instance type actually used in the backend, or is it randomly generated? I am thinking if it is randomly generated it might be better to keep more readable : )

Comment (mjibril) [reply catalogs/v5/fluidstack/vms.csv:2]: All instances are available in the backend. FluidStack supports custom/configurable plans, but SkyPilot supports fixed plans. So a couple of configurable plans were added as fixed in Skypilot in order to support more accelerators.

File: .github/workflows/update-fluidstack-catalog.yml (hunk 1 above)
Comment (mjibril) [reply .github/workflows/update-fluidstack-catalog.yml]: Auto updater workflow is removed.

**Review by Michaelvll — APPROVED** [review]
Thanks for adding the fluidstack catalog @mjibril! LGTM.
